infrastructure/kube/keep-test/monitoring/prometheus-pvc.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: prometheus-pvc
```

Could we drop `-pvc` suffix?

```suggestion
  name: prometheus
```